elastic-stack/Dockerfile

```diff
@@ -21,10 +29,22 @@ RUN \
   apt-get install -y kibana && \
 # Install Logstash
   apt-get install -y logstash && \
+# Install S3 Repository Plugin
+  /usr/share/elasticsearch/bin/elasticsearch-plugin install -b repository-s3 && \
 # Allow network access to Elasticsearch from the host
   sed -i "s/#network.host:.*/network.host: 0.0.0.0/" /etc/elasticsearch/elasticsearch.yml && \
   sed -i "s/#server.host:.*/server.host: 0.0.0.0/" /etc/kibana/kibana.yml
 
+RUN \
+# Set the repository-s3 configuration
+  chmod 0644 /etc/cron.d/elasticsearch_backup && \
```

Comment:
My opinion about this is that all the backup process should be done outside the Docker container. All of this stuff adds complexity on something which should be simple: run the elastic stack.